src/element/bounds.ts

```diff
@@ -457,7 +506,20 @@ const getLinearElementRotatedBounds = (
 // We could cache this stuff
 export const getElementBounds = (
   element: ExcalidrawElement,
-): [number, number, number, number] => {
+  recompuate = false,
+): Bounds => {
+  if (!recompuate) {
+    const cachedBounds = cachedElementBounds.getCachedBounds(element);
+    if (
+      cachedBounds &&
+      cachedBounds.version === element.version &&
+      cachedBounds.versionNonce === element.versionNonce &&
```

Comment:
using either `version` or `versionNonce` should be fine (`version` is an incrementing integer, `versionNonce` is a random number — used for collab reconciliation purposes but should work fine).

Reply:
also, I'd encapsulate the check into the `getCachedBounds` class itself so that if we ever end up using it elsewhere, we don't have to duplicate the logic (and it's safer).

Reply:
@dwelle  Make sense!